index.js

```diff
@@ -95,6 +95,20 @@ function getNewVersionCode(programOpts, versionCode, versionName, resetBuild) {
 	return versionCode ? versionCode + 1 : 1;
 }
 
+/**
+ * CFBundleShortVersionString must be a string composed of three period-separated integers.
+ * @private
+ * @param {String} versionName The full version string
+ * @return {String} e.g. returns '1.2.3' for given '1.2.3-beta.1'
+ */
+function getCFBundleShortVersionString(versionName) {
+	const array =
+		versionName && typeof versionName === "string"
+			? versionName.match(/\d*\.\d*.\d*/g)
+			: [];
+	return array[0] ? array[0] : versionName;
```

Comment:
match returns null if no match is found.  Should have a test for that and I think this ternary needs to change